### PR TITLE
Romanian translation - strings.xml

### PR DESCRIPTION
**Pull request by AlexUnderHood**

### CODE REVIEW

File: app/src/main/res/values-ro/strings.xml [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+    <string name="send_sweep_hint">Șterge</string>
+    <string name="send_generate_paymentid_hint">Generează</string>
+    <string name="send_qr_hint">Scanează</string>
+    <string name="send_prepare_hint">Prepgătește</string>
+    <string name="send_dispose_hint">Dispune (Anulează)</string>
+    <string name="send_send_label">Spend my sweet Moneroj</string>
+    <string name="send_send_timed_label">Spend my sweet Moneroj (%1$s)</string>
+    <string name="send_really_send_hint">Ultimul pas - Confirmă!</string>
+    <string name="send_qr_invalid">Cod QR invalid</string>
+    <string name="send_qr_address_invalid">Cod QR de plată invalid/string>
```
Comment (m2049r) [inline app/src/main/res/values-ro/strings.xml:265]: ```/string>``` => ```</string>```

File: app/src/main/res/values-ro/strings.xml [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+    </string-array>
+
+    <string-array name="priority">
+        <item>Prioritate Implicită</item>
+        <item>Prioritate mică</item>
+        <item>Prioritate medie</item>
+        <item>Prioritate mare</item>
+    </string-array>
+
+    <string-array name="currency" translatable="false">
```
Comment (m2049r) [inline app/src/main/res/values-ro/strings.xml:365]: please remove all strings with ```translatable="false"```

**Comment by m2049r** [comment]
please rebase to current master - we have some new strings (in ```strings.xml```) and text in ```help.xml```. sorry about that.

**Review by m2049r — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
Pinged @AlexUnderHood on reddit. Pinging also here

**Comment by m2049r** [comment]
@AlexUnderHood following translations are missing

- menu_changepw
- info_crazypass_enabled
- changepw_progress
- changepw_failed
- changepw_success
- prompt_changepw
- prompt_changepwB
- prompt_fingerprint_auth
- bad_fingerprint
- generate_fingerprint_hint
- generate_fingerprint_warn
- generate_bad_passwordB
- generate_empty_passwordB
- generate_crazypass_label

**Comment by erciccione** [comment]
still no answer from @AlexUnderHood . @m2049r Could you fix and recover this as we talked on IRC?

**Comment by erciccione** [comment]
ping @m2049r

**Comment by m2049r** [comment]
next release will pick up all pending translations.

**Pull request merged**